lint_test/analysis_options.yaml

```
@@ -11,7 +11,8 @@ custom_lint:
     - function_lines_of_code:
       max_lines: 50
     - avoid_non_null_assertion
-    - avoid_late_keyword
+    - avoid_late_keyword:
+      allow-initialized: true
```

The convention for the proj is to use snake-case.
```suggestion
      allow_initialized: true
```